### PR TITLE
Handle sticky topic param when there is no data for that topic

### PR DESCRIPTION
**Pull request by shifucun**

### CODE REVIEW

File: static/js/place/subtitle.tsx [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ */
+
+import React from "react";
+
+interface SubtitlePropsType {
+  category: string;
+  dcid: string;
+}
+
+class Subtitle extends React.Component<SubtitlePropsType, unknown> {
```
Comment (beets) [inline static/js/place/subtitle.tsx:24]: please add a component description (and can we call this PageSubtitle to be more specific?)

**Review by beets — CHANGES_REQUESTED** [review]
nice change!

File: static/js/place/subtitle.tsx (hunk 1 above)
Comment (shifucun) [reply static/js/place/subtitle.tsx]: Done

**Review by beets — APPROVED** [review]
thanks for making the change!

**Pull request merged**